package.json

```diff
@@ -21,56 +21,56 @@
   "homepage": "https://github.com/metamaps/metamaps#readme",
   "dependencies": {
     "ajaxq": "0.0.7",
-    "attachmediastream": "1.4.2",
-    "autolinker": "1.4.0",
-    "babel-cli": "6.22.2",
-    "babel-loader": "6.2.10",
+    "attachmediastream": "2.0.0",
+    "autolinker": "1.4.3",
+    "babel-cli": "6.26.0",
+    "babel-loader": "7.1.2",
     "babel-plugin-lodash": "3.2.11",
-    "babel-plugin-transform-class-properties": "6.22.0",
-    "babel-preset-es2015": "6.22.0",
-    "babel-preset-react": "6.22.0",
+    "babel-plugin-transform-class-properties": "6.24.1",
+    "babel-preset-es2015": "6.24.1",
+    "babel-preset-react": "6.24.1",
     "backbone": "1.3.3",
-    "clipboard-js": "0.3.2",
-    "commonmark": "0.27.0",
-    "csv-parse": "1.1.10",
-    "emoji-mart": "0.3.7",
-    "getscreenmedia": "2.0.0",
+    "clipboard-js": "0.3.5",
+    "commonmark": "0.28.1",
+    "csv-parse": "1.2.1",
+    "emoji-mart": "1.0.1",
+    "getscreenmedia": "4.0.1",
     "hark": "1.1.5",
-    "howler": "2.0.2",
-    "jquery": "3.1.1",
-    "json-loader": "0.5.4",
+    "howler": "2.0.4",
+    "jquery": "3.2.1",
+    "json-loader": "0.5.7",
     "lodash": "4.17.4",
-    "node-uuid": "1.4.7",
+    "node-uuid": "1.4.8",
     "outdent": "0.3.0",
-    "react": "15.4.2",
-    "react-dom": "15.4.2",
-    "react-draggable": "2.2.3",
-    "react-dropzone": "3.9.1",
-    "react-onclickoutside": "5.9.0",
-    "react-router": "3.0.2",
-    "redux": "3.6.0",
-    "riek": "1.0.7",
-    "simplewebrtc": "2.2.2",
+    "react": "15.6.1",
+    "react-dom": "15.6.1",
+    "react-draggable": "3.0.3",
+    "react-dropzone": "4.1.2",
+    "react-onclickoutside": "6.5.0",
+    "react-router": "4.2.0",
+    "redux": "3.7.2",
+    "riek": "1.1.0",
+    "simplewebrtc": "3.0.1",
     "socket.io": "1.3.7",
-    "webpack": "2.2.1"
+    "webpack": "3.5.6"
```

Comment:
woahhhh test

Reply:
seems to work, and apparently it's an easy upgrade per https://medium.com/webpack/webpack-3-official-release-15fd2dd8f07b